### PR TITLE
feat: build the library also in cjs to support project with jest / not fully support ESM

### PR DESCRIPTION
**Pull request by GiovanniVisentiniCasavo**
## What

I enable the build of cjs project

## Why

not all project are ready to support ESM only library. 
I think that we should enable the use of the old cjs format to made the dev exprirence easy, without requiring to change build system / test library when start using this library

### CODE REVIEW

**Comment by moebiusmania** [comment]
**non blocking for this time**, but next time please check https://github.com/casavo/habitat/blob/main/CONTRIBUTING.md#how-to-submit-code-changes

File: vite.config.ts
Comment (marcel-casavo) [inline vite.config.ts:20]: I would use this guide here to property enable a cjs build for the lib https://vitejs.dev/guide/build.html#library-mode

Comment (moebiusmania) [reply vite.config.ts:20]: @marcel-casavo do you mean the parts about the `package.json` entry points? because we are already using the library mode

Comment (marcel-casavo) [reply vite.config.ts:20]: exactly, aligning what is missing with the intended purpose(to have a cjs build)

**Comment by moebiusmania** [comment]
@pietrograndi this can have impacts on your #59 PR

**Review by marcel-casavo — APPROVED** [review]

**Comment by marcel-casavo** [comment]
@GiovanniVisentiniCasavo LGTM but would wait for multiple approvals before merging as we all need to by aligned/aware of this change

**Review by moebiusmania — APPROVED** [review]

**Pull request merged**
